lochness/utils/source_check.py

```diff
@@ -289,7 +289,13 @@ def check_source(Lochness: 'lochness') -> None:
 
         # box
         print('Loading data list from BOX')
-        box_files_df = collect_box_files_info(keyring)
+        tmp_box_db = Path(Lochness['phoenix_root']) / \
+                '.tmp_box_source_files.csv'
```

Comment:
Is this file created out of this function and a prior?

Reply:
`collect_box_files_info` takes about 10 mins to run, which was too slow for me to test the related functions in the `source_check.py`. So I'm saving the output of the `collect_box_files_info` for me to use in testing, by providing `test=True` to the `check_source`. It's there only for testing purposes.

Reply:
Again, is this file created before the code reaches line 292?

Reply:
No- it would be not there on the first run.

Reply:
Then wouldn't this line run into error on the first run?

Reply:
Yes, only if someone is manually providing `test=True` input to the `source_check.py` function. (this would be me) I added this for myself to check the processes following the `collect_box_files_info`

Default usage would be `test=False`